doc/2/guides/Asset Category/index.md

```diff
@@ -0,0 +1,74 @@
+---
+code: false
+type: page
+title: Assets Category
+description: Asset Category document
+order: 110
+---
+
+
+# Assets
+
+An assetCategory document represents a category of assets. 
+
+An assetCategory contain metadata information about assets of the category. The value of the metadata can be given by the asset, or in some case, by the assetCategory, if all assets of the category have the same value for the metadata. 
+
+An assetCategory can have a parent and, like with Object-oriented programming, it inherit caracteristics of its parent.
+
+You can have a clean and complete representation of an assetCategory using the get function of the assetCategory Controller 
+
+**Example:** _Clean representation of an assetCategory given by a get_
+```js
+{
+        "name": "example",
```

Comment:
This example should be aligned with 2 spaces

Reply:
done